ghost/core/core/server/data/migrations/versions/5.5/2022-07-21-08-56-add-jobs-table.js

```diff
@@ -0,0 +1,11 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('jobs', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    name: {type: 'string', maxlength: 191, nullable: false, unique: true},
+    status: {type: 'string', maxlength: 50, nullable: false, defaultTo: 'queued', validations: {isIn: ['started', 'finished', 'failed', 'queued']}},
```

Comment:
Even though it's not important here, `isIn` requires the value to be an array of an array - like you have in the schema